lib/main.dart

```diff
@@ -69,26 +80,14 @@ class _PangolinState extends State<Pangolin> {
       if (Pangolin.settingsBox.get("language").toString().length < 5) {
```

Comment:
remove the whole getLangFromHive function and the method call just below, basically remove the initState method from this class

Reply:
done

Reply:
done